css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,136 @@
+$progress-bar-background-color: $secondary-background-glow-high-contrast !default;
+$progress-bar-value-background-color: $primary !default;
+$progress-bar-value-background-color-warning: $warning !default;
+$progress-bar-value-background-color-danger: $danger !default;
+$progress-bar-value-background-color-success: $success !default;
+$progress-bar-height: $spacer-2 !default;
+$progress-bar-height-sm: $spacer-1 !default;
+$progress-bar-border-radius: $border-radius-rounded !default;
+$progress-bar-indeterminate-animation-duration: 2s !default;
+
+.progress-bar {
+	display: block;
+	position: relative;
+	width: 100%;
+	max-width: 100%;
+	height: $progress-bar-height;
+	padding: 0;
+	border: none;
+	border-radius: $progress-bar-border-radius;
+	background-color: $progress-bar-background-color;
+	appearance: none;
```

Comment:
I wonder if this one is necessarily. I tried removing it on Edge, Chrome, and Firefox and didn't notice any difference.

Reply:
It's more a "just in case" scenario, we have it assigned on other components too.